lib/cef.js

```
@@ -11,19 +11,20 @@
  *
  * Please see the README.md file for usage.
  */
+"use strict";
 
-const os = require('os');
-const syslog = require('./syslog');
-const Formatter = require('./formatter');
```

Yes, this is because of the use of `strict` mode.  This is mostly a habit of mine, I just push all languages into strict mode out of habit. Consider nodejs only runs on V8 and not just any javascript engine, I think the only real payoff here is well, more strictness.  I have a preference for throwing errors on bad syntax instead of javascript silently eating the error.

On the other hand, I only really care about adding heka transport support to node-cef, so if this is a deal breaker, I'll defer to your judgement.